templates/matching.jsx

```diff
@@ -74,6 +76,16 @@ export default function Matching(props) {
 
               </div>
 
+              {_canShowCorrectness &&
+              <div
+                key={`answer-${_index}`}
+                className="matching-item__answer-container"
+                dangerouslySetInnerHTML={{
+                  __html: (_isInteractionComplete && _canShowCorrectness && _correctAnswers) || '&nbsp;'
```

Comment:
Minor: This `_canShowCorrectness` check is already part of the conditional block render so is not necessary here